src/v8n.js

```diff
@@ -40,16 +40,16 @@ function proxyContext(context) {
 }
 
 function proxyContextEs5(context) {
-  const buildRuleSet = ruleSet =>
+  const addRuleSet = (ruleSet, targetContext) =>
     Object.keys(ruleSet).forEach(prop => {
-      context[prop] = (...args) => {
-        const newContext = proxyContextEs5(context._clone());
+      targetContext[prop] = (...args) => {
+        const newContext = proxyContextEs5(targetContext._clone());
         return newContext._applyRule(ruleSet[prop], prop)(...args);
       };
     });
```

Comment:
The `addRuleSet` function should return the changed context after adding all the rules.